samples/apps/smallbank/clients/small_bank_client.cpp

```diff
@@ -46,7 +46,7 @@ class SmallBankClient : public Base
       json j;
       j["name"] = to_string(i);
       const auto response =
-        json::from_msgpack(conn->call("SmallBank_balance", j));
+        json::from_msgpack(conn->call("users/SmallBank_balance", j));
```

Comment:
Should this prefixing be done in `RpcTlsClient::call`? We already have the appropriate prefix in the `sni` member of `TlsClient`

Reply:
It depends how we want to test stuff, but it certainly could.